### PR TITLE
Support nullable value in sha2 func

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

Summary about this PR

## Changelog

 
- Improvement
 
 
## Related Issues

#2881 

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by sundy-li** [comment]
cc @mshauneu

**Review by zhyass — APPROVED** [review]
LGTM

File: common/datavalues/src/arrays/builder.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+use common_arrow::arrow::bitmap::Bitmap;
```
Comment (zhyass) [inline common/datavalues/src/arrays/builder.rs:1]: Maybe move to line 17 is better

Comment (sundy-li) [reply common/datavalues/src/arrays/builder.rs:1]: Ok, trivial fix.

**Pull request merged**